packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-svg.tsx

```diff
@@ -82,120 +192,97 @@ export class ScDialSvg {
+            strokeWidth={this.sizeConfig.dialThickness}
+            labelSize={this.sizeConfig.labelSize}
+            iconSize={this.sizeConfig.iconSize}
+            loadingText={this.loadingText}
           />
-        )}
-        {this.point ? (
-          <text
-            x={valueBottomMiddleX}
-            y={valueBottomMiddleY}
-            font-size={this.sizeConfig?.fontSize}
-            font-weight="bold"
-            text-anchor="middle"
-          >
-            <tspan>
-              {this.stream?.unit ? round(this.point?.y as number) : value}
-              <tspan font-size={this.sizeConfig?.unitSize}>{this.unit}</tspan>
-            </tspan>
-          </text>
         ) : (
-          <text
-            x={valueBottomMiddleX}
-            y={valueBottomMiddleY}
-            font-size={this.sizeConfig?.fontSize}
-            font-weight="bold"
-            text-anchor="middle"
-            fill={ColorConfigurations.SECONDARYTEXT}
-          >
-            {NO_VALUE_PRESENT}
-          </text>
-        )}
-
-        {showDetailedName ? (
-          <text
-            x={circleX}
-            y={labelBottomMiddleY}
-            font-weight="bold"
-            font-size={this.sizeConfig?.labelSize}
-            text-anchor="middle"
-            fill={ColorConfigurations.PRIMARYTEXT}
-          >
-            {this.stream?.detailedName}
-          </text>
-        ) : null}
-
-        {showLabel && (
-          <text
-            x={this.labelBottomMiddleX}
-            y={labelBottomMiddleY}
-            font-size={this.sizeConfig?.labelSize}
-            font-weight="bold"
-            text-anchor="start"
-            fill={labelColor || ColorConfigurations.PRIMARYTEXT}
+          <svg
+            width="100%"
+            height="100%"
+            viewBox="0 0 276 276"
+            data-testid="current-value"
+            preserveAspectRatio="xMidYMin meet"
           >
-            {label}
-          </text>
-        )}
-
-        {showLabel && (
-          <g transform={`matrix(1,0,0,1,${this.iconLeftTopX},${this.iconLeftTopY})`}>
-            {icon && getIcons(icon, labelColor, this.sizeConfig?.iconSize)}
-          </g>
+            <g transform={`matrix(1,0,0,1,${DIAMETER},${DIAMETER})`}>
+              <path
+                d={this.defaultRing}
+                fill={ColorConfigurations.GRAY}
+                stroke={ColorConfigurations.WHITE}
+                stroke-linejoin="round"
+              />
+              <path d={this.ColorRing} fill={labelColor} stroke={ColorConfigurations.WHITE} stroke-linejoin="round" />
+            </g>
+            {this.point ? (
+              <text
+                x={valueBottomMiddleX}
+                y={valueBottomMiddleY}
+                font-size={this.size?.fontSize}
+                font-weight="bold"
+                text-anchor="middle"
+              >
+                <tspan>
+                  {this.stream?.unit ? round(this.point?.y as number) : value}
```

Comment:
should utilize the `significantDigit` input - currently `significantDigit` input is only considered when displaying percentages